table/table.go

```diff
@@ -245,3 +245,23 @@ var TableFromMeta func(allocators autoid.Allocators, tblInfo *model.TableInfo) (
 
 // MockTableFromMeta only serves for test.
 var MockTableFromMeta func(tableInfo *model.TableInfo) Table
+
+// CachedTable is a Table, and it has a UpdateLockForRead() method
+// UpdateLockForRead() according to the reasons for not meeting the read conditions, update the lock information,
+// And at the same time reload data from the original table.
+type CachedTable interface {
+	Table
+
+	// LoadDataFromOriginalTable Load data from the original table to the cache
+	LoadDataFromOriginalTable(ctx sessionctx.Context) error
+
+	// GetMemCache Get the MemBuffer used by the cache table
+	GetMemCache() kv.MemBuffer
+
+	// IsReadFromCache Check if the cache table is readable
+	IsReadFromCache(ts uint64) bool
```

Comment:
You can merge the two methods to one `TryGetMemCache(ts uint64) (kv.MemBuffer, bool)`

Reply:
yeah i will do it.